include/deal.II/base/utilities.h

```diff
@@ -302,6 +302,21 @@ namespace Utilities
    *
    * Use this class as in <code>fixed_int_power@<5,2@>::%value</code> to
    * compute 5<sup>2</sup>.
+   *
+   * @deprecated This template has been deprecated in favor of C++11's support
+   * for <code>constexpr</code> calculations, e.g., use
+   *
+   * @code
+   * constexpr int value = Utilities::pow(2, dim);
+   * @endcode
+   *
+   * instead of
+   *
+   * @code
+   * const int value = Utilities::fixed_int_power<2, dim>::value;
+   * @endcode
+   *
+   * to obtain a constant expression for <code>value</code>.
```

Comment:
Can you ` DEAL_II_DEPRECATED` so that people get a warning when they are using the function

Reply:
Good point. I fixed it.